p2p/pex/pex_reactor.go

```diff
@@ -216,6 +216,13 @@ func (r *PEXReactor) logErrAddrBook(err error) {
 	}
 }
 
+func (r *PEXReactor) InitPeer(peer Peer) Peer {
+	id := string(peer.ID())
+	r.requestsSent.Delete(id)
+	r.lastReceivedRequests.Delete(id)
+	return peer
+}
+
 // RemovePeer implements Reactor.
 func (r *PEXReactor) RemovePeer(p Peer, reason interface{}) {
 	id := string(p.ID())
```

Comment:
same here. why do we need to do the same thing in `RemovePeer` (I'm talking about `r.requestsSent.Delete(id)` and `r.lastReceivedRequests.Delete(id)`)